workers/unity/Packages/com.improbable.gdk.gameobjectcreation/GameObjectCreationSystemHelper.cs

```diff
@@ -1,20 +1,26 @@
 using Improbable.Gdk.Core;
 using Unity.Entities;
+using UnityEngine;
 
 namespace Improbable.Gdk.GameObjectCreation
 {
     public static class GameObjectCreationSystemHelper
     {
-        public static void EnableStandardGameObjectCreation(World world)
+        public static void EnableStandardGameObjectCreation(World world, IEntityGameObjectCreator creator = null,
```

Comment:
We still need to update the `WorkerUtils` file as part of this PR right? Otherwise we are not using any `IEntityGameObjectCreator` anymore by default.

Reply:
I think it adds one in the function but I agree this looks odd. Slightly better to keep this as two overloads I think.

Reply:
If you read below, I'm using null as a special value to add the default parametrized by the stuff from the World. It's confusing, so I'll revert back to 2 methods instead

Reply:
Ah sorry, didn't read until the end.